frontend/src/layout/navigation-3000/components/TopBar.tsx

```diff
@@ -228,7 +229,7 @@ function Breadcrumb({ breadcrumb, here, isOnboarding }: BreadcrumbProps): JSX.El
     } else {
         nameElement = (
             <span className="flex items-center gap-1.5">
-                {breadcrumbName || <i>Unnamed</i>}
+                {breadcrumbName === '' ? <em>Unnamed</em> : breadcrumbName}
```

Comment:
Can it ever be `null`? I dont think `breadcrumb`s are required

Reply:
from the props I don't think so but will check nonetheless

Reply:
![CleanShot 2025-05-23 at 21 30 20@2x](https://github.com/user-attachments/assets/e1b3ab68-01bc-47ec-a8eb-5e985bbeb73c)

i don't think it can, the name property is always mandatory